assertj-tests/assertj-integration-tests/assertj-core-kotlin/pom.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.assertj</groupId>
+    <artifactId>assertj-integration-tests</artifactId>
+    <version>3.24.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>assertj-core-kotlin</artifactId>
+
+  <name>AssertJ Core Integration Tests - Kotlin</name>
+
+  <properties>
+    <kotlin.version>1.7.10</kotlin.version>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.jetbrains.kotlin</groupId>
```

Comment:
Very cosmetic: would you mind keeping the group ID alphabetical order among dependencies? Like:

org.assertj // org.jetbrains.kotlin // org.junit.jupiter